README.md

```diff
@@ -1,6 +1,6 @@
 # Trulioo SDK for Java #
 
-## Version 1.0.2.1
+## Version 1.0.2.2
```

Comment:
Since it is not a bug fix, I think it should be 1.0.3.0 but up to you

Reply:
It is updated to 1.0.3